src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/Repository/ReleaseVersionRepositoryTests.cs

```diff
@@ -210,27 +118,26 @@ public class GetLatestReleaseVersionTests : ReleaseVersionRepositoryTests
         [Fact]
         public async Task Success()
         {
-            var publications = _dataFixture
+            var (publication1, publication2) = _dataFixture
```

Comment:
wonder if to bother adding two publications? Maybe just add 1 test that checks to see that the publication Guid passed into the method we're testing grabs the right one, where we do set-up 2 publications, but then for the other tests just leave it as 1 publication to make it leaner?

Reply:
Great suggestion, that certainly leans things up quite a bit!

I did this everywhere so now each inner class of tests has one method beginning with the name `MultiplePublications_` which still uses the `GenerateTuple2()` to create two publications, and tests the data comes back from the specified publication.

In all other cases the tuple of publications is removed.

I also means we get rid of all the `.ForIndex(0,`, `.ForIndex(1,` etc. 😄